Modules/Server/Sources/PocketCastsServer/Public/Sync/BackgroundSyncManager.swift

```diff
@@ -53,7 +53,7 @@ public class BackgroundSyncManager: NSObject {
         // we also need to perform all these on the same URLSession as download tasks, so the app can be put to sleep and woken up when they are done later
 
         let urlSession = createUrlSession(identifier: BackgroundSyncManager.sessionIdPrefix + UUID().uuidString)
-        guard let token = KeychainHelper.string(for: ServerConstants.Values.syncingV2TokenKey), let refreshTask = refreshDownloadTask(subscribedPodcasts: subscribedPodcasts, urlSession: urlSession), let upNextTask = upNextDownloadTask(token: token, urlSession: urlSession), let syncTask = syncDownloadTask(token: token, urlSession: urlSession) else {
+        guard let token = try? KeychainHelper.string(for: ServerConstants.Values.syncingV2TokenKey), let refreshTask = refreshDownloadTask(subscribedPodcasts: subscribedPodcasts, urlSession: urlSession), let upNextTask = upNextDownloadTask(token: token, urlSession: urlSession), let syncTask = syncDownloadTask(token: token, urlSession: urlSession) else {
             FileLog.shared.addMessage("Unable to create tasks required to perform background refresh")
```

Comment:
I was wondering if seeing this message in the console should be part of the testing steps? On my tests I wasn't able to see it 🤔

Reply:
I believe this is only called on watchOS. I'm not sure how to simulate that background extension.

Reply:
Oh, you're right.